dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -249,13 +249,8 @@ private Field dbSaveUpdate(final Field throwAwayField) throws DotDataException {
       final List<String> takenFieldVars = fieldsAlreadyAdded.stream().map(Field::variable).collect(
               Collectors.toList());
 
-      // check if the field variable is compatible with GraphQL
-      if(!GraphQLUtil.isVariableGraphQLCompatible(throwAwayField)) {
-          takenFieldVars.add(throwAwayField.name());
-      }
+      String tryVar = getFieldVariable(throwAwayField, takenFieldVars);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'tryVar' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)